_includes/values.inc

```diff
@@ -15,3 +15,4 @@
 {% assign cybozuLink = '//cybozu.com' %}
 {% assign redHatLink = '//www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation' %}
 {% assign upboundLink = '//upbound.io' %}
+{% assign cloudRaftLink = '//cloudraft.io' %}
```

Comment:
is there perhaps a link to your site that is more specific to Rook/Ceph? that could be more beneficial to folks coming to this page.

Reply:
Sure, I have pointed to our Kubernetes page.